test/test_distributions.py

```diff
@@ -1452,46 +1462,66 @@ def test_laplace_shape_tensor_params(self):
 
 
 class TestKL(TestCase):
+
     def setUp(self):
+        # These are pairs of distributions with 4 x 4 paramters as specified.
+        # The first of the pair e.g. bernoulli[0] varies column-wise and the second
+        # e.g. bernoulli[1] varies row-wise; that way we test all param pairs.
+        bernoulli = pairwise(Bernoulli, [0.1, 0.2, 0.6, 0.9])
+        beta = pairwise(Beta, [0.5, 2.0, 0.5, 2.0], [0.3, 0.3, 3.0, 3.0])
+        chi2 = pairwise(Chi2, [0.2, 1.0, 2.0, 5.0])
+        exponential = pairwise(Exponential, [0.1, 0.5, 2.0, 10.0])
+        gamma = pairwise(Gamma, [0.5, 2.0, 0.5, 2.0], [0.5, 0.5, 2.0, 2.0])
+        gumbel = pairwise(Gumbel, [-2.0, 2.0, -2.0, 2.0], [0.5, 2.0, 0.5, 2.0])
+        laplace = pairwise(Laplace, [-2.0, 2.0, -2.0, 2.0], [0.5, 2.0, 0.5, 2.0])
+        normal = pairwise(Normal, [-2.0, 2.0, -2.0, 2.0], [0.5, 2.0, 0.5, 2.0])
+        pareto = pairwise(Pareto, [-2.0, 2.0, -2.0, 2.0], [0.5, 2.0, 0.5, 2.0])
+        uniform_interval = pairwise(Uniform, [0, 0, 0.4, 0.8], [1, 0.2, 0.6, 1])
+        uniform_positive = pairwise(Uniform, [1, 1.5, 2, 4], [1.2, 2.0, 3, 7])
+        uniform_real = pairwise(Uniform, [-2, -1, 0, 2], [-1, 1, 1, 4])
+        dirichlet = pairwise(Dirichlet, [[0.1, 0.2, 0.7],
+                                         [0.5, 0.4, 0.1],
+                                         [0.33, 0.33, 0.34],
+                                         [0.2, 0.2, 0.4]])
+        self.sampling_extra = [1000, 500, 1500, 1000, 2500, 500, 2000, 10000, 5000,
```

Comment:
This is a lot of werd numbers. I'd simplify to something monotonic and embed it in the for loop below
```py
for samples in [1000, 3000, 10000, 30000, 100000]:
```

Reply:
Sure. I will take care of this in my next commit.